### DISCUSSION TITLE
Any solution to protect customer content? (something like DRM)

### DISCUSSION THREAD

**Discussion by sorasit-teebeea** [mozilla/hubs#4899] [Hubs Cloud Questions] [answered]
First, this question is already asked in mozilla hubs discord.

Answered by mattbcool:
`Making video's unable to be copied is not a feature of hubs cloud. I'm not aware of any solutions for you there.`

I re-post here for direct team support.

**Background**
Currently, we are developing hub cloud based on customer customization.
Our customer is concern about content that will use to play in VR room is able to download, copy and screen recording.

**Question**
Is there any solution to protect customer content that will use to display on hubs?

content maybe video. but, I ask for flexibility of other content (like images, glb, gltf or any 3d object file)

**Comment by matthewbcool** [comment, marked as answer]
To answer your question again- No, this is not a feature. This kind of system would be tricky to implement and probably anything you did people could figure out a way around it.  You can make a feature request here: https://github.com/mozilla/hubs/issues/new?assignees=&labels=enhancement%2C+jira-hubs&template=feature_request.md&title=
